flaml/training_log.py

```diff
@@ -159,6 +161,8 @@ def training_log_writer(filename: str):
         w = TrainingLogWriter(filename)
         w.open()
         yield w
+    except FileNotFoundError:
```

Comment:
We can remove this except block as the error message below doesn't provide more detailed information than the builtin error message.

Reply:
I got it. It seems reasonable. Do I need to update another commit to modify it?

Reply:
Yes. Please update with another commit. Thanks!

Reply:
I updated. Thanks!